src/actions/errors.ts

```diff
@@ -22,7 +22,7 @@ export function dismissError(index: number): ActionFunc {
     };
 }
 
-export function getLogErrorAction(error: ErrorObject, displayable = false) {
+export function getLogErrorAction(error: any, displayable = false) {
```

Comment:
Setting this to `error: serializeError.ErrorObject` fails because v2.1.0 of `serializer-error` doesn't export ErrorObject, best I can tell.

Could someone with stronger TypeScript fu than me perhaps guide how to create/export an explicit type definition that matches the structure of `ErrorObject` below?

```
export type ErrorObject = {
	name?: string;
	stack?: string;
	message?: string;
	code?: string;
}
```